source/partials/_footer.html.erb

```diff
@@ -0,0 +1,37 @@
+<footer class="footer">
+  <div class="footer__content">
+
+    <div class="footer__column footer__column--insiter">
+      <h2 class="footer__logo">Insiter<span>.</span></h2>
+      <p>
+        You inside the web.
+        <br>
+        Endereço do Insiter Goiânia
```

Comment:
What do you think about using 2 `<p>` here instead of using a `<br>`?

Reply:
Good idea, i'll change